Intersect.Client.Core/Interface/Game/EventWindow.cs

```diff
@@ -119,56 +83,62 @@ private EventWindow(Canvas gameCanvas, Dialog dialog) : base(gameCanvas, nameof(
             Dock = Pos.Fill,
             Font = _defaultFont,
             FontSize = 12,
-            Padding = new Padding(8),
+            Padding = new Padding(12),
         };
 
-        _promptPanel.SizeToChildren(recursive: true);
+        CreateOptionButtons();
 
-        #region Configure and Display
+        // Load JSON to apply layout/templates
+        try
+        {
+            Name = nameof(EventWindow);
+            LoadJsonUi(GameContentManager.UI.InGame, Graphics.Renderer.GetResolutionString());
```

Comment:
Event window lacks usable fallback layout when there is no JSON in the GUI.

Many of the windows that are already more up-to-date from the last times Panda and I were in this race are all styled before the `loadjsonui` call. This is so that if the user doesn't have the JSON (and with this pull, many won't), they don't need to download it from the official assets. Instead, it will be created from the default style styled in the code. Then, `loadjsonui` will load the user's styling. The assets shouldn't have been merged; the JSON sent to the assets should be a "pure" JSON created by the code styling and not customized manually.

Reply:
updated!
now the code will create it's own json file with a default styled layout as fallback if there's no json file

<img width="1365" height="762" alt="{72D7F9F9-A405-49EF-BAFC-1EB66AD62A51}" src="https://github.com/user-attachments/assets/a59d778d-fa91-489d-a24d-7f791d8880af" />

Reply:
( also: deleted the merged EventWindow.json from the assets repo )